VimeoNetworking/Sources/Request.swift

```diff
@@ -181,7 +181,14 @@ public struct Request<ModelType: MappableResponse>
             }
         }
         
-        return components!.string!
+        if let theURI = components?.string?.stringByRemovingPercentEncoding
```

Comment:
Nit - Should we `guard` at the top of the functions? So this `?` on `components` would disappear 💭